learn-ocaml-client.opam.locked

```diff
@@ -32,7 +32,7 @@ depends: [
   "bigarray-compat" {= "1.0.0"}
   "bigstringaf" {= "0.8.0"}
   "biniou" {= "1.2.1"}
-  "cmdliner" {= "1.0.4"}
+  "cmdliner" {= "1.1.0"}
```

Comment:
Finally: even if it's wise to keep `"cmdliner" {>= "1.1.1"}` in the `*.opam` files, could you please bump the version in `*.opam.locked` files while you are on it?
```suggestion
  "cmdliner" {= "1.1.1"}
```

Reply:
sorry I've just edited my previous comment, that was wrong. I obviously meant "even if it's wise to keep `"cmdliner" {>= "1.1.0"}`.